src/OrchardCore/OrchardCore.ContentManagement.Abstractions/IContentPickerResultProvider.cs

```diff
@@ -7,6 +7,8 @@ namespace OrchardCore.ContentManagement
     public interface IContentPickerResultProvider
     {
         string Name { get; }
+
+        Task<string> GetContentPickerItemDescription(ContentItem contentItem, string pattern, string defaultValue);
```

Comment:
Don't change this interface, or do anything to do with descriptions here.

Reply:
removed title and description from this interface